test/e2e/tools_bucket_web_test.go

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) The Thanos Authors.
+// Licensed under the Apache License 2.0.
+
+package e2e_test
+
+import (
+	"net/http/httptest"
+	"testing"
+	"github.com/cortexproject/cortex/integration/e2e"
+	"github.com/thanos-io/thanos/pkg/objstore/client"
+	"github.com/thanos-io/thanos/pkg/objstore/s3"
+	e2edb "github.com/cortexproject/cortex/integration/e2e/db"
+	"github.com/thanos-io/thanos/pkg/testutil"
+	"github.com/thanos-io/thanos/test/e2e/e2ethanos"
```

Comment:
```suggestion
	"net/http/httptest"
	"testing"

	"github.com/cortexproject/cortex/integration/e2e"
	e2edb "github.com/cortexproject/cortex/integration/e2e/db"

	"github.com/thanos-io/thanos/pkg/objstore/client"
	"github.com/thanos-io/thanos/pkg/objstore/s3"
	"github.com/thanos-io/thanos/pkg/testutil"
	"github.com/thanos-io/thanos/test/e2e/e2ethanos"
```

We follow some rules for import-order to stay consistent. Separate standard lib, external and thanos imports.

Reply:
PS: Don't commit this suggestion directly from GitHub UI as DCO will fail for that commit :)

Reply:
addressed all your changes! 🙂